<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TabWindow</class>
 <widget class="QWidget" name="TabWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>626</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QTextEdit {	border: 0;}QComboBox {	font: 10pt \&quot;Segoe UI\&quot;;    color: #555;	border: 1px solid #ced4da;}QComboBox::drop-down {    border: 0px;}QComboBox::down-arrow {	image: url(:/images/arrow.svg);	width: 10px;	height: 10px;}QComboBox::on {	border: 4px solid #c2dbfe;}QComboBox QListView {	font-size: 12px;	border: 1px solid rgba(0,0,0,10%);	padding: 5px;		background-color: #fff;	outline: 0px;}QComboBox QListView::item {	font: 10pt \&quot;Segoe UI\&quot;;	padding-left: 10px;	background-color: #fff;	}QComboBox QListView::item {		background-color: #1e90ff;}QComboBox QListView::item::selectd {	background-color: #1e90ff;}QTableView {	border: 0px;}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QTextEdit" name="textEdit_"/>
     <widget class="QWidget" name="widget" native="true">
      <property name="styleSheet">
       <string notr="true">margin: 0;
padding: 0;</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QComboBox" name="comboBox_">
         <property name="styleSheet">
          <string notr="true">margin: 0;
padding: 0;</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QTableView" name="tableWidget_">
         <property name="styleSheet">
          <string notr="true">QScrollBar:horizontal {
    border: 2px solid grey;
    background: #32CC99;
    height: 15px;
    margin: 0px 20px 0 20px;
}
QScrollBar::handle:horizontal {
    background: white;
    min-width: 20px;
}
QScrollBar::add-line:horizontal {
    border: 2px solid grey;
    background: #32CC99;
    width: 20px;
    subcontrol-position: right;
    subcontrol-origin: margin;
}

QScrollBar::sub-line:horizontal {
    border: 2px solid grey;
    background: #32CC99;
    width: 20px;
    subcontrol-position: left;
    subcontrol-origin: margin;
}


QScrollBar:vertical {
	min-width: 20px;
    border: 2px solid grey;
    background: #32CC99;
    height: 15px;
    margin: 0px 20px 0 20px;
}
QScrollBar::handle:vertical {
    background: white;
    min-width: 20px;
	min-height: 20px;
}
QScrollBar::add-line:vertical {
    border: 2px solid grey;
    background: #32CC99;
    width: 20px;
    subcontrol-position: right;
    subcontrol-origin: margin;
	min-height: 20px;
}

QScrollBar::sub-line:vertical {
    border: 2px solid grey;
    background: #32CC99;
    width: 20px;
    subcontrol-position: left;
    subcontrol-origin: margin;
}</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
